src/components/PersonalData.js

```diff
@@ -2,37 +2,22 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 import { connect } from "react-redux";
 import { Field, reduxForm } from "redux-form";
-
 import Form from "reactstrap/lib/Form";
-// import TextInput from "components/EduIDTextInput";
 import EduIDButton from "components/EduIDButton";
 import CustomInput from "../login/components/Inputs/CustomInput";
-
-// import "style/Emails.scss";
-// import "style/PersonalData.scss";
-// import "style/DashboardMain.scss";
-
 import "../login/styles/index.scss";
 
 /* FORM */
 
-const validatePersonalData = (values) => {
+const validatePersonalData = (values, props) => {
   const errors = {};
-  const spacePattern = /^\s+$/;
-  const withSpecialCharacters  = /[`!€%&?~#@,.<>;':"\/\[\]\|{}()-=_+]/;
-
-  ["given_name", "surname", "display_name", "language"].forEach((pdata) => {
-    if (!values[pdata]) {
-      errors[pdata] = "required";
-    }
-    //add space pattern instead of trim because the input datas will be added without space.
-    else if(spacePattern.test(values[pdata])){
-      errors[pdata] = "required";
+  ["given_name", "surname", "display_name", "language"].forEach((inputName) => {
+    if (!values[inputName] || !values[inputName].trim()) {
```

Comment:
When I add spaces before or after a saved value I expect to get an error (and to see the "required" text), but the input is green and I am able to save the value. Have I misunderstood this code?

Reply:
I have updated (using empty string regex pattern)